.changeset/strange-steaks-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-internal/benchmarks': patch
```

Comment:
nit: it doesn't matter either way but we've not been consistent about doing changeset for internal/private packages. i generally do empty changeset for these since it's not end-user facing, but it doesn't really hurt to keep them. maybe it's good to have a record of changes even for non-published packages in the changelog file.

Reply:
I'm happy to switch this to empty.  I wasn't sure about internal packages 🤷 I'm on team augustjk for now since I would still be clicking rerun-failed-jobs if you didn't tell me to bump this LOL.

Reply:
lol i tried to edit in place and lint check wasted time complaining about my markdown formatting so i'll just leave it with the patch note.